variables.tf

```diff
@@ -242,12 +242,19 @@ variable "s3_region" {
   default     = ""
 }
 
-variable "allow_list" {
+variable "ingress_allow_list" {
   type        = list(string)
-  description = "List of CIDRs we allow to access the infrastructure"
+  description = "List of CIDR blocks we allow to access the infrastructure"
   default     = []
 }
 
+variable "egress_allow_list" {
+  type        = list(string)
+  description = "List of CIDR blocks we allow the infrastructyre to access"
+  default     = ["0.0.0.0/0"]
```

Comment:
the complexity to figure out any limitation of this is very high, allow talking to all, if an end user needs to limit, they have the context of what explicit CIDR blocks they need access to for their use case.

Reply:
Yeah, that's definitely on them to figure out.